recipes/isl/build.sh

```diff
@@ -0,0 +1,10 @@
+if [ "$(uname)" = 'Darwin' ]
+then
+    export CC="clang"
+    export CXX="clang++"
```

Comment:
Add this [line]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L7 ) for Mac. This [line]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L17 ) for Linux.